src/GitTools.jl

```diff
@@ -75,8 +75,15 @@ setprotocol!(proto::Union{Nothing, AbstractString}=nothing) = GIT_PROTOCOL[] = p
 # TODO: extend this to more urls
 function normalize_url(url::AbstractString)
     m = match(GITHUB_REGEX, url)
-    (m === nothing || GIT_PROTOCOL[] === nothing) ?
-        url : "$(GIT_PROTOCOL[])://github.com/$(m.captures[1]).git"
+    if m === nothing || GIT_PROTOCOL[] === nothing
+        url
+    else
+        if GIT_PROTOCOL[] == "ssh"
+            "git@github.com:$(m.captures[1]).git"
```

Comment:
You can also use `ssh://git@github.com/` prefix which makes this a bit more consistent with all the other protocols.

Reply:
but less consistent with what a user expects to see / github suggests you to use